Mathlib/Algebra/Polynomial/Eval/Defs.lean

```diff
@@ -538,6 +539,10 @@ protected theorem map_ofNat (n : ℕ) [n.AtLeastTwo] :
 theorem map_dvd (f : R →+* S) {x y : R[X]} : x ∣ y → x.map f ∣ y.map f :=
   _root_.map_dvd (mapRingHom f)
 
+--TODO rename to `map_dvd_map`
```

Comment:
I don't understand this todo, and either way is this intended to be merged into mathlib?

Reply:
Whoops, that's copypasta